code/platformio.ini

```diff
@@ -1,5 +1,5 @@
 [platformio]
-default_envs = nodemcu-lolin
+default_envs = yagusmart-touch-hwmod-1g
```

Comment:
```suggestion
default_envs = nodemcu-lolin
```

Reply:
Of course. Sorry I didn't look closely enough